toolchest_client/api/status.py

```diff
@@ -43,7 +43,7 @@ class Status(str, Enum):
     FAILED = "failed"
 
 
-class ThreadStatus(str, Enum):
+class PrettyStatus(str, Enum):
```

Comment:
Any reason this is `PrettyStatus` and not just `Status`. I understand the pretty print function having that name but the status to me can just be `Status`.

Reply:
The `Status` enum above `PrettyStatus` corresponds 1:1 with API-side statuses. The `PrettyStatus` enum (formerly `ThreadStatus` was previously for tracking thread statuses.

There wasn't much logic tied to the `PrettyStatus/ThreadStatus` except for pretty-printing status (e.g. just `executing`, no `awaiting_execution` / `beginning_execution` / etc).

I think that PrettyStatus should be removed – or exist as a mapping from Status to a printable status – but I kept it in place to avoid making too extensive of a refactor. Open to removing it in this PR, though.

Reply:
Ah didn't expand the file and see `Status` above. I would be okay leaving the refactor for later especially since multi-value enums in python aren't great.